action.yml

```diff
@@ -100,6 +100,15 @@ inputs:
   plan:
     description: 'Where to either save an Update Plan or read an Update Plan from'
     required: false
+  log-verbosity:
+    description: 'Log verbosity of CLI during execution 1-10; Anything >3 is very verbose'
+    required: false
+  log-to-std-err:
+    description: 'Log to stderr instead of to files'
+    required: false
```

Comment:
Is there any value in logging to files inside the action? Unless users have setup to trawl /tmp at the end of the job their just going to get lost? Wonder if this be flipped relative to automationapi and be "log-to-files" and default to logging to stderr?

Reply:
@ekbduffy would you consider settling for log-verbose and debug options and revisit this log-to-std-err at a later stage?

Reply:
Yup the others make sense, just --log-to-stderr should probably default on if either of the others are set.